packages/lab/src/menu/Menu.tsx

```diff
@@ -0,0 +1,88 @@
+import {
+  ComponentPropsWithoutRef,
+  CSSProperties,
+  forwardRef,
+  SyntheticEvent,
+  useMemo,
+  // useState,
+} from "react";
+import { MenuContext } from "./MenuContext";
+import { useControlled, useFloatingUI } from "@salt-ds/core";
+import { flip, offset, shift, limitShift, Placement } from "@floating-ui/react";
+
+export interface MenuProps extends ComponentPropsWithoutRef<"div"> {
+  open?: boolean;
+  defaultOpen?: boolean;
+  placement?: Placement;
+  onOpenChange?: (event: SyntheticEvent, newOpen: boolean) => void;
+}
+
+export const Menu = forwardRef<HTMLDivElement, MenuProps>(function Menu(
+  props,
+  ref
+) {
+  const {
+    children,
+    defaultOpen,
+    open,
+    placement = "bottom-start",
+    onOpenChange,
+    ...rest
+  } = props;
+  // const { submenu } = useMenuContext();
+
+  const [openState, setOpenState] = useControlled({
+    controlled: open,
+    default: Boolean(defaultOpen),
+    name: "ListControl",
+    state: "open",
+  });
+
+  const setOpen = (event: SyntheticEvent, newOpen: boolean) => {
+    setOpenState(newOpen);
+    onOpenChange?.(event, newOpen);
+  };
+
+  // const [activeState, setActiveState] = useState<string | undefined>(undefined);
+
+  // const setActive = (id?: string) => {
+  //   if (id) {
+  //     setActiveState(id);
+  //   } else {
+  //     setActiveState(undefined);
+  //   }
+  // };
+
+  // const hasParent = submenu !== undefined;
```

Comment:
do we still need these - there's a couple of commented out bits in various files